learn/experimental/autobatching.md

```diff
@@ -0,0 +1,34 @@
+# Auto-batching
+
+If enabled, this feature automatically combines multiple document addition requests ([`POST`](/reference/api/documents.md#add-or-replace-documents) or [`PUT`](/reference/api/documents.md#add-or-update-documents)) into batches to speed up indexation.
```

Comment:
Should we mention `batchUid` here? And add a sample response?

Reply:
Good idea, I will briefly mention `batchUid`. I'm not sure if it's necessary to add a sample response though since it's supposed to be a short, bare-bones guide, not full documentation 🤔